flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -49,6 +62,22 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
     channel.setMethodCallHandler(null)
   }
 
+  override fun onAttachedToActivity(binding: ActivityPluginBinding) {
+    activity = binding.activity
+  }
+
+  override fun onDetachedFromActivity() {
+    // Stub
```

Comment:
```suggestion
    activity = nul
```

Maybe we clean the reference or make `activity` a WeakRef?